content/examples/config/config.docker.yaml.md

```diff
@@ -1,27 +1,14 @@
-```yaml
-# See detailed configuration settings: https://www.pomerium.com/docs/reference/
+```yaml title="config.yaml"
+signing_key: LS0tLS1CRUdJTiBFQyBQUklWQVRFIEtFWS0tLS0tCk1IY0NBUUVFSURMV3Q3ZkczV2ZkYjk5elFHQTJObEJXcCt3d0c1aGJoR3MzY29JUlo2SjRvQW9HQ0NxR1NNNDkKQXdFSG9VUURRZ0FFcGtRRktLUUdqcVdzbDlYYkUwWmZLL2ZhbHJ2NENWSWtqSTlydXlCbHdOeDYzNmhZRnBtKwpNM0llTXNUKzRreExidVlZSGZDeUtjQzFnZ1BjSWpCYktRPT0KLS0tLS1FTkQgRUMgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
Is this accurate?

Reply:
I used the command in our [signing key](https://pomerium.com/reference/signing-key) reference page to generate this key. We also did something similar in our Enterprise Quickstart.